content/en/docs/v3.6/op-guide/configuration.md

```diff
@@ -164,6 +164,10 @@ Flags are presented below using the format `--flag-name DEFAULT_VALUE`.
   Comma-separated whitelist of origins for CORS, or cross-origin resource sharing, (empty or * means allow all).
 --host-whitelist '*'
   Acceptable hostnames from HTTP client requests, if server is not secure (empty or * means allow all).
+--tls-min-version 'TLS12'
+  Minimum TLS version supported by etcd. Possible values: TLS12, TLS13.
+--tls-max-version ''
+  Maximum TLS version supported by etcd. Possible values: TLS12, TLS13 (empty will be auto-populated by Go).
```

Comment:
Same here, we should remove `possible values` so that we don't need to maintain updates in the future. Leaving, `Empty will be auto-populated by Go.` is fine.